lib/uwb/include/uwb/UwbRegisteredCallbacks.hxx

```diff
@@ -30,21 +40,21 @@ struct UwbRegisteredSessionEventCallbacks
      * @param session The session for which the event occurred.
      * @param reason The reason the session ended.
```

Comment:
This comment should be updated since OnSessionEnded is now a `std::weak_ptr`. Same with the other event callbacks

Reply:
updated docs